policy-staging.json

```diff
@@ -0,0 +1,43 @@
+{
+  "instance_min_count": 1,
+  "instance_max_count": 3,
+  "scaling_rules": [
+    {
+      "metric_type": "throughput",
+      "breach_duration_secs": 60,
+      "threshold": 10,
```

Comment:
This number is totally arbitrary,
Other metrics would likely be more useful but we're not seeing enough variation yet I reckon.

I think we should take a look at grafana whilst we're scaling down across the next weeka dn see if there's a better suggestion here?

Reply:
How about scaling based on `responsetime` instead of `throughput`?  Using `throughput` means we'll scale linearly with the number of requests, even if the existing instances are keeping up.  Scaling when things start to slow down makes more sense to me.

Reply:
Yeah @bevanloon and I were chatting about that one, I think you'd recommended that earlier too?
We got a bit caught up on the idea of this being an average and if that conceals a bit too much?

For instance if we moved MFA to synchronous calls, we might not mind if the response time got a little longer there, but perhaps we'd want really right times on login / registration?

Not sure how the average might either skew what's going on or cover over differences per path?